src/test/java/org/isf/dicom/TestFileSystemDicomManager.java

```diff
@@ -40,19 +32,28 @@
 import org.isf.dicom.service.DicomIoOperations;
 import org.isf.dicomtype.model.DicomType;
 import org.isf.dicomtype.service.DicomTypeIoOperationRepository;
-import org.isf.dicomtype.test.TestDicomType;
+import org.isf.dicomtype.TestDicomType;
 import org.isf.menu.manager.Context;
 import org.isf.utils.exception.OHDicomException;
 import org.isf.utils.exception.OHException;
 import org.isf.utils.exception.OHServiceException;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.ApplicationContext;
 import org.springframework.util.FileSystemUtils;
 
+import java.io.File;
+import java.io.FileReader;
+import java.lang.reflect.Field;
+import java.text.ParseException;
+import java.util.Properties;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
the project standard is to put static imports first.

Reply:
Sorry forgot to add the [OpenHospital-code-style-configuration.xml](https://github.com/informatici/openhospital-core/blob/develop/.ide-settings/idea/OpenHospital-code-style-configuration.xml) , should be fixed